ci-operator/config/open-cluster-management/security-middleware/open-cluster-management-security-middleware-main.yaml

```diff
@@ -0,0 +1,36 @@
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: open-cluster-management
+    tag: nodejs10-linux-amd64
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: unit
+  commands: |
+    mkdir /tmp/npmcache
+    export npm_config_cache=/tmp/npmcache
+    export SELF="make -f Makefile.prow"
+    make -f Makefile.prow unit-test
+  container:
+    from: src
+- as: sonarcloud
+  commands: |
```

Comment:
Why not having just one make command here instead?

Reply:
We use a consolidated build harness for all 55+ of our repos instead of a different make file for each repo. Some repos need to make minor changes to the build harness settings or run multiple steps from the build harness.